sabnzbd/interface.py

```diff
@@ -273,6 +273,8 @@ def log_warning(txt):
     if not (cfg.disable_key() or nokey):
         key = kwargs.get('apikey')
         if not key:
+            key = kwargs.get('session')
```

Comment:
There is a security 'gap' with the new HTML login form because CherryPy doesn't force basic auth for the TAPI. So in https://github.com/sabnzbd/sabnzbd/pull/551/commits/69b6d5fead8491953b50692b3e43af214681bf64 I switched the TAPI to also use ```check_apikey```. Otherwise if the user disables his API key but has username/password turned on, there is no check for username/password because ```check_session``` assumes CherryPy's basic auth handles it.

But because I switched it, all the Config pages broke because they send a ```session=xxx``` instead of an ```apikey=xxx```.